backend/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "babel-node --presets env,stage-2 index.js"
+  },
+  "author": "CreativeIT",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.18.3",
```

Comment:
do we have to use ^ for version, why just without and hard-coded. What's the best practise